drivers/lcz_fem/Kconfig

```diff
@@ -28,14 +28,55 @@ config LCZ_FEM_INTERNAL_ANTENNA
 	default $(dt_node_bool_prop,$(DT_NODELABEL_LCZ_FEM),internal-antenna)
 
 config LCZ_FEM_REGION
```

Comment:
Could we call attention to these in the repo readme?

If some information was available that describes the different ways a project can be set up (kconfig, board file changes / overlays) would be good to address queries raised by Chris L.

Reply:
I'll add a link to the instructions in the repo readme